book/chapters/config.py

```diff
@@ -1,210 +1,19 @@
-        else:
-            ds = ServiceXSourceXAOD(sx_ds_name, backend=sx_backend_name)
-
-    # TODO: If we run Overlap Removal, then we must have a PV in the event. Unfortunately,
-    # we do not yet know how to filter out events without a PV in them. So we turn off OR
-    # by default for now. When this is fixed, remove the warning in the calibration notebook.
-    ds = calib_tools.query_update(ds, perform_overlap_removal=False)
-
-    return ds
-
-
-# Build the individual samples. First, the DAOD_PHYS samples, which use default
-# calibrations.
-ds_zee_r21 = make_ds(_samples["zee_r21"])
-ds_zee_untyped_r21 = make_ds(_samples["zee_untyped_r21"])
-ds_ztautau_r21 = make_ds(_samples["ztautau_r21"])
-ds_bphys_r21 = make_ds(_samples["bphys_r21"])
-ds_ttbar_r22 = make_ds(_samples["ttbar_r22"])
-
-# Turns out the muon events don't all have a primary vertex, which is
-# required to do overlap removal. So for now we'll turn that off until
-# we understand what the proper thing to do here is.
-# ds_zmumu_r21 = calib_tools.query_update(
-#     make_ds(_samples["zmumu_r21"]),
-#     perform_overlap_removal=False
-# )
-
-# To demonstrate some features of jets, we need an older R21 sample (which contains
-# topo clusters). Default calibration is different for this guy.
-# ds_jz2_exot15_r21 = calib_tools.query_update(
-#     make_ds(_samples["jz2_exot15_r21"]),
-#     jet_collection="AntiKt4EMTopoJets",
-#     jet_calib_truth_collection="AntiKt4TruthJets",
-#     perform_overlap_removal=False,
-# )
-
-
-def match_eta_phi(jets, jets_to_match) -> ak.Record:
-    """Match `jets_to_match` to the `jets` given. There will always be
-    at least one jet found.
-
-    The awkward array needs to have leaves called `pt`, `eta`, and `phi`.
-
-    Args:
-        jets (_type_): Source jets
-        jets_to_match (_type_): Jets to match to `jets`
-
-    Returns:
-        _type_: Matched jets 1:1 in `jets` from `jets_to_match`.
-    """
-
-    to_match_pt = jets_to_match.pt
-    to_match_eta = jets_to_match.eta
-    to_match_phi = jets_to_match.phi
-    jet_eta = jets.eta
-    jet_phi = jets.phi
-
-    pair_eta = ak.cartesian([jet_eta, to_match_eta], axis=1, nested=True)
-    pair_phi = ak.cartesian([jet_phi, to_match_phi], axis=1, nested=True)
-
-    delta_eta = np.abs(pair_eta[:, :, :]["0"] - pair_eta[:, :, :]["1"])
-    # TODO: Missing wrap around fro phi
-    delta_phi = np.abs(pair_phi[:, :, :]["0"] - pair_phi[:, :, :]["1"])
-
-    delta = delta_eta ** 2 + delta_phi ** 2
-
-    # TODO: remove anything larger that 0.2*0.2
-    best_match = ak.argmin(delta, axis=2)
-
-    return ak.Record(
-        {
-            "eta": to_match_eta[best_match],
-            "phi": to_match_phi[best_match],
-            "pt": to_match_pt[best_match],
-        }
-    )
+from servicex import deliver, dataset
+
+def get_dataset(jet_info_per_event):
+    """Sends request for dataset to servicex backend. Pass in FuncADLQueryPHYSLITE and returns list of files in dataset."""
```

Comment:
Call this "query" or similar, rather than jet info per event - that juts happen to be what you are doing now - it will be tracks, etc., as you move on.